.github/workflows/release.yml

```diff
@@ -0,0 +1,18 @@
+name: Main
+
+on:
+    push:
+        tags:
+            - "v*.*.*"
```

Comment:
I think this will trigger if anyone in the organisation pushes a tag to any branch. Can we be stricter and somehow require that the tags are on the `main` branch? If not, I'll see whether I can disallow tag pushes from org members.